pages/github/fork.tsx

```diff
@@ -22,15 +24,72 @@ export default function Authorizing() {
     }
   }
 
+  const [forkValidating, setForkValidating] = useState(true)
+ 
+  useEffect(() => {  
+    const branch = "master";
+    fetch(`/api/proxy-github`, {
+      method: 'POST',
```

Comment:
method 'POST', but proxy method 'GET'

Reply:
I get `TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.` when I use 'GET' on the proxy.